jsonnet/components/kube-state-metrics.libsonnet

```
@@ -212,11 +241,18 @@ function(params)
                           cpu: '2m',
                         },
                       },
-                      volumeMounts: [{
-                        mountPath: '/tmp',
-                        name: tmpVolumeName,
-                        readOnly: false,
-                      }],
+                      volumeMounts: [
+                        {
+                          mountPath: '/tmp',
+                          name: tmpVolumeName,
+                          readOnly: false,
+                        },
+                        {
```

(nit) a comment would clarify why the configmap is always mounted
```suggestion
                        // The custom resource state configmap is always mounted in the kube-state-metrics container and only when the VPA CRD is installed, CMO will add `--custom-resource-state-config-file` to the container arguments list.
                        {
```